### ISSUE TITLE
When using model.BBGroup1.Group2.Group3 etc...  Anything after group 4 is ignored by Action Wheel

### ISSUE THREAD

**Issue by Jadestrouble** [Blancworks/Figura#137]
Sorry if I miss-use some terms, I'm not very experienced in this.

I was trying to make an emote folder inside the head folder in Blockbench, so that the emotes would sync up to whatever the head does, when I tried specifying the emote 
`action_wheel.SLOT_2.setFunction(function() model.Axolotl.Head.Emotes["Shades"].setEnabled(true) end)`
it didn't work, when I changed it to 
`action_wheel.SLOT_2.setFunction(function() model.Axolotl.Head.Emotes["Shades"].setEnabled(true) end)`
it worked, but it enabled all emotes, not the single one I wanted it to. I decided to put the Shades Blockbench Shape into another folder so I could simplify with 
`action_wheel.SLOT_2.setFunction(function() model.Axolotl.Head.Emotes.Shades.setEnabled(true) end)`
This didn't work, I should probably specify that there were no errors in chat, it just didn't do/change anything I thought it might of been something with the word "Shades" or Something with the specific folder, but I did some testing and the only way I got it to work was by removing the emotes folder and making it
'action_wheel.SLOT_2.setFunction(function() model.Axolotl.Head.Shades.setEnabled(true) end)'
not a complete deal breaker but it did make things more complicated. Below are two files that should display what I'm taking about

I'm not sure if this is an intentional feature, or a technical limitation, or just a bug, but it would be nice if it was changed
[Figura Doesnt Work.zip](https://github.com/Blancworks/Figura/files/6757453/Figura.Doesnt.Work.zip)
[Figura Works.zip](https://github.com/Blancworks/Figura/files/6757454/Figura.Works.zip)

**Comment by TheOneTrueZandra** [comment]
`model.Axolotl.Head.Emotes.Shades.setEnabled(true)`
and
`model.Axolotl.Head.Emotes["Shades"].setEnabled(true)`

are effectively the same code, to lua. There should be no functional difference with them on a language level, so this is weird...

**Comment by Jadestrouble** [comment]
yeah it is weird. seems to be an issue with how many blockbench groups inside of blockbench groups it can detect. ill have to try and make sure the bug is consistent and not some weird fluke with my code or blockbench.

**Comment by Jadestrouble** [comment]
Ok, I've been able to replicate it consistently, here's what I did.
**Step 1.** Create New Blockbench model
I made mine 6x6x6, and above the players head for simplicity sake
**Step 2.** Make some groups,
I used 5 as it was the lowest I could go and still encounter the bug, 6+ groups also probably experience this.
looks like this in blockbench
![image](https://user-images.githubusercontent.com/81522136/124346850-07f4e780-db96-11eb-8d2e-6c0f53e81fab.png)
**Step 3.** Do the rest on the stuff in order to make a model (file sytem)
**Step 4.** Paste in the code
` action_wheel.SLOT_1.setFunction(function() model.Group1.Group2.Group3.Group4.Group5.setRot({0,45,0}) end)`
**Step 5** Use Slot 1 of the action wheel, the model does not rotate 45 degrees as it should. If you, move the model in blockbench to Group 4 and change the code to
` action_wheel.SLOT_1.setFunction(function() model.Group1.Group2.Group3.Group4.setRot({0,45,0}) end)`
it works perfectly

again I have NO IDEA what might cause this and it could be user error on my part. The zip that I did to test this out is below
[Bug Showcase.zip](https://github.com/Blancworks/Figura/files/6758047/Bug.Showcase.zip)
